package.json

```diff
@@ -71,6 +71,7 @@
 		"bootstrap": "^5.1.3",
 		"express": "^4.17.1",
 		"helmet": "^4.6.0",
+		"moment": "^2.29.1",
```

Comment:
We don't need the `moment` library so let's remove it and uninstall it

Reply:
Ok Coung, deleted and unistalled